src/entities/appointment.entity.ts

```diff
@@ -14,11 +14,11 @@ export default class Appointment {
   @PrimaryGeneratedColumn()
   id: number;
 
-  @Column()
+  @Column({ default: 'pending' })
```

Comment:
will the status field transition to other values additional to 'pending'? and why define `pending` as default? In any case, I think this should be an enum field type.

Enums help you to identify which are the possible values for a given field also help as documentation to know which values are acceptable here and if you receive something different take the proper actions throwing an error